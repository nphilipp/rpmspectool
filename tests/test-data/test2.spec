# An entirely too short spec file

Name: test
Version: 1
Release: 1
Summary: Test spec file

License: CC-0
URL: https://github.com/some/example/repo
Source0: %{url}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
Patch0: %{url}/commit/0123456789abcdef0123456789abcdef01234567.patch
