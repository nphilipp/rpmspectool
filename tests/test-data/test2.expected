Source0: https://github.com/some/example/repo/archive/v1.tar.gz#/test-1.tar.gz
Patch0: https://github.com/some/example/repo/commit/0123456789abcdef0123456789abcdef01234567.patch
